### PR TITLE
scaffold Fast USDC contract

### PR DESCRIPTION
**Pull request by turadg**
in preparation for other tickets

## Description

Sets up the basic structure of the fast-usdc contract.

### Security Considerations
none

### Scaling Considerations
none

### Documentation Considerations
none

### Testing Considerations
none

### Upgrade Considerations
none

### CODE REVIEW

File: packages/fast-usdc/proposals/start-fast-usdc.js [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+    /** @type {BootstrapManifest} */
+    manifest: {
+      [startFastUsdc.name]: {
+        consume: {
+          chainStorage: true,
+          chainTimerService: true,
+          localchain: true,
+          cosmosInterchainService: true,
+
+          // limited distribution during MN2: contract installation
```
Comment (0xpatrickdev) [inline packages/fast-usdc/proposals/start-fast-usdc.js]: > // limited distribution during MN2: contract installation

curious - what's meant by this comment?

File: packages/fast-usdc/proposals/start-fast-usdc.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+    privateArgs,
+  };
+
+  const { instance } = await E(startUpgradable)(startOpts);
+
+  produceInstance.reset();
+  produceInstance.resolve(instance);
+  trace('startFastUsdc done', instance);
+};
+harden(startFastUsdc);
```
Comment (0xpatrickdev) [inline packages/fast-usdc/proposals/start-fast-usdc.js]: nit: maybe create `PoolShares` here? since the contract expects it and the test setup contains it

**Review by 0xpatrickdev — APPROVED** [review]

File: packages/fast-usdc/proposals/start-fast-usdc.js (hunk 1 above)
Comment (turadg) [reply packages/fast-usdc/proposals/start-fast-usdc.js]: anticipating MN3 in which anyone can install

Comment (turadg) [reply packages/fast-usdc/proposals/start-fast-usdc.js:124]: anticipating MN3 in which anyone can install

File: packages/fast-usdc/proposals/start-fast-usdc.js (hunk 2 above)
Comment (turadg) [reply packages/fast-usdc/proposals/start-fast-usdc.js]: Good point. The fact that CI is green without doing that highlights that this module isn't tested at all. I'll leave it out until we get to it, in milestone 2.

**Pull request merged**